src/set.js

```diff
@@ -0,0 +1,103 @@
+'use strict'
+
+export default class Set {
+  constructor(initialValues=[]) {
+    this.elements = initialValues
+  }
+
+  add(value) {
+    if (this.elements.indexOf(value) === -1) {
+      this.elements[this.elements.length] = value
+      return this.elements
+    }
+    return false
+  }
+
+  isEmpty() {
+    if (this.elements.length === 0 ){
+      return true
+    }
+    return false
+
+
+  }
+
+  contains(value) {
+    if (this.elements.indexOf(value) > 0) {
+      return true
+    }
+    return false
+  }
+
+  remove(value) {
+    if (this.elements.indexOf(value) > 0) {
+      return this.elements.splice(this.elements.indexOf(value),1)
+    }
+    return "The value to remove does not exist"
+  }
+
+  forEach(func) {
+    this.elements = this.elements.map(func)
```

Comment:
this would be better

```js
forEach(iterator) {
  this.elements.forEach(iterator)
  return this
}
```

This would be better for your learning

```js
forEach(iterator) {
  for (let i = this.length - 1; i >= 0; i--) {
    iterator(this.elements[i], i)
  }
  return this
}
```